### PR TITLE
Vultr Inventory script

### PR DESCRIPTION
**Pull request by OpenSourceZombie**
##### COMPONENT NAME

contrib/inventory/vultr.py
##### ANSIBLE VERSION

```
ansible 2.3.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = Default w/o overrides
```
##### SUMMARY

Added vultr dynamic inventory script

```
ansible -i vultr.py server.domain.com -m ping
```

### CODE REVIEW

File: contrib/inventory/vultr.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python
+'''
+Ansible Inventory Script
+
+Makes some API calls to api.vult.com/v1 to get information about servers and dumps them as json
```
Comment (mscherer) [inline contrib/inventory/vultr.py:5]: There is a typo (api.vult.com should be api.vultr.com)

**Comment by alikins** [comment]
This and  https://github.com/ansible/ansible/pull/19386 seem to overlap.

**Comment by patran** [comment]
Is PR going to be added to the official release?

**Comment by resmo** [comment]
closing in favor of #44400

close_me

**Pull request closed without merging**